doc/modules/decomposition.rst

```diff
@@ -30,6 +30,9 @@ strong assumptions on the isotropy of the signal: this is for example
 the case for Support Vector Machines with the RBF kernel and the K-Means
 clustering algorithm.
 
+Note: the :class:`PCA` object centers the input data for each feature before
+applying the SVD.
```

Comment:
Maybe move this to the beginning of the previous paragraph (and remove "Note" and "class..") ? .e.g 
``
PCA centers [...] before applying the SVD. The optional parameter whiten etc..
```

Reply:
Done